test/parallel/test-tls-ecdh.js

```diff
@@ -1,31 +1,31 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
```

Comment:
Nit: I think it's ok to move the `assert` require after the `common.hasCrypto` check.

I'm pointing this out because other tests do this and it would be nice for consistency.
Thanks.